pom.xml

```diff
@@ -69,6 +69,8 @@
         <yarn.arguments>build:production</yarn.arguments>
         <require-capability>osgi.extender;filter:="(osgi.extender=org.jahia.bundles.blueprint.extender.config)"</require-capability>
         <jahia-module-signature>MCwCFC9dqq924zUTQuE1WmbmW+G78ZZNAhQ/EEuk6kwcs4xMDIi7/XLf1gjkCA==</jahia-module-signature>
+        <import-package>graphql.annotations.annotationTypes;version="[6.1,9)"</import-package>
```

Comment:
is it still useful if we update the parent to 8.1.0.0 and gql-provider  to 2.7.0 ?

Reply:
yes I believe so, if we decide to move annotations all the way up to 8x later and avoid recompile

Reply:
ok sorry just revisiting your slack message about this one. I guess we can take this out for now and we'll just rebuild again if we ever decide to move annotations up to 8x.

Reply:
Removed in last commit